vmdk_plugin/main.go

```diff
@@ -38,7 +38,7 @@ const (
 	pluginSockDir = "/run/docker/plugins"
 	mountRoot     = "/mnt/vmdk" // VMDK and photon volumes are mounted here
 	photonDriver  = "photon"
-	vmdkDriver    = "vmdk"
+	vmdkDriver    = "vsphere"
```

Comment:
Looks like driverName is case sensitive. 
```
docker volume create -d VMDK --name=v1@bad
Error response from daemon: create v1@bad: create v1@bad: Error looking up volume plugin VMDK: plugin not found
```

"vSphere" with uppercase S is error prone. Can we register multiple times - "vsphere", "vSphere", "VSPHERE". Or a PR to Docker to make plugin name case insensitive?